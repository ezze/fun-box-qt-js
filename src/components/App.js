import React from 'react';

import Sidebar from './Sidebar';
import Map from './Map';

const App = () => (
    <div className="app">
        <Sidebar />
        <Map />
    </div>
);

export default App;
