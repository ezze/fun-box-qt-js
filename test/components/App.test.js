import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import App from '../../src/components/App';

describe('application component', () => {
    it('rendering', () => {
        const renderer = new ShallowRenderer();
        renderer.render(<App />);
        const tree = renderer.getRenderOutput();
        expect(tree).toMatchSnapshot();
    });
});
