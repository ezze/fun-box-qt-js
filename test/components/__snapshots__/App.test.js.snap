// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`application component rendering 1`] = `
<div
  className="app"
>
  <Sidebar />
  <Connect(Map) />
</div>
`;
